GccBoardProject1/src/main.c

```diff
@@ -92,28 +74,11 @@ int main (void) {
 		if (!ge_is_sleeping()) {
 			// Send frame
 			uint8_t buffer[512];
-			
-			//double ambient_temp = get_ambient_temp();
-			
-			//uint16_t len = snprintf((char *) buffer, sizeof(buffer), "It is %.2lf degrees Celsius.\r\n", ambient_temp);
-			//uart_out(buffer, len);
+
 			ge_get_frame(PIXEL_BUFFER);
-			uint16_t size = 0;
-			for (int i = 0; i < NUM_PIXELS; i++) {
-				size += sprintf((char *) (buffer + size), "%d,", PIXEL_BUFFER[i]);
-			}
-			buffer[size-1] = '\r';
-			buffer[size] = '\n';
-			frameCount++;
-			if (frameCount <= NUM_RAW_FRAMES) {
-				initialize_mem(PIXEL_BUFFER, frameCount);
-			} else {
-				dstack(MEDIAN_FILTER_POINTERS_BUFF, PIXEL_BUFFER);;
-			}
-			uint16_t *frame_median = median_frame(MEDIAN_FILTER_POINTERS_BUFF, frame1);
-			
-			count = countPeople(&count, frame_median, frameCount, COUNT_BUFF, lastFrame);
-			uart_out(buffer, size+1);
+			start_count(frame_count, PIXEL_BUFFER, MEDIAN_FILTER_POINTERS_BUFF, COUNT_BUFF);
+			frame_count++;
```

Comment:
I don't think this belongs here. It should be be up to main to keep track of the number of frames.